### PR TITLE
S2U-5 T&Qs: dynamic rubrics

### PR DESCRIPTION
**Pull request by bgarciaentornos**

### CODE REVIEW

**Comment by bgarciaentornos** [comment]
See https://github.com/sakaiproject/sakai/pull/11766

Created as a draft, as I want to check if this column should be not-null

File: docs/conversion/s2u_features/S2U-5-oracle.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- S2U-5 --
+ALTER TABLE rbc_rubric ADD adhoc NUMBER(1) DEFAULT 0;
```
Comment (mpellicer) [inline docs/conversion/s2u_features/S2U-5-oracle.sql:2]:
```suggestion
ALTER TABLE rbc_rubric ADD adhoc NUMBER(1, 0) DEFAULT 0 NOT NULL;
```

File: docs/conversion/s2u_features/S2U-5-mysql.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- S2U-5 --
+ALTER TABLE rbc_rubric ADD adhoc bit(1) DEFAULT 0;
```
Comment (mpellicer) [inline docs/conversion/s2u_features/S2U-5-mysql.sql:2]:
```suggestion
ALTER TABLE rbc_rubric ADD adhoc bit(1) NOT NULL DEFAULT 0;
```

File: docs/conversion/s2u_features/S2U-5-oracle.sql (hunk 1 above)
Comment (bgarciaentornos) [reply docs/conversion/s2u_features/S2U-5-oracle.sql:2]: But it can be null, all the existing entries will be null...

**Review by mpellicer — APPROVED** [review]

**Pull request merged**